src/System.Windows.Forms/src/System/Windows/Forms/Application.cs

```diff
@@ -37,9 +38,16 @@ public sealed partial class Application
     private static bool s_comCtlSupportsVisualStylesInitialized;
     private static bool s_comCtlSupportsVisualStyles;
     private static FormCollection? s_forms;
-    private static readonly object s_internalSyncObject = new();
+    private static readonly Lock s_internalSyncObject = new();
     private static bool s_useWaitCursor;
 
+#pragma warning disable WFO5001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+    private static SystemColorMode? s_systemColorMode;
+#pragma warning restore WFO5001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
Not a priority but have this comment twice is distracting
```suggestion
#pragma warning restore WFO5001
```

Reply:
Let's purge them in .NET 10.